astropy/io/ascii/tests/test_rst.py

```diff
@@ -147,6 +147,10 @@ def test_trailing_spaces_in_row_definition():
     1    4.5  bar
    ==== ==== ====  
 """  # noqa
+    # make sure no one accidentally deletes the trailing whitespaces in the
```

Comment:
While we're add this, could we perhaps ensure that auto-trailing-space deleting editors do not mess up the example? E.g., the following make the same input, but have quote marks at the end (also removing the need for `noqa`):
```
    table = ("""\n"""
"""# comment (with blank line above)\n"""
"""   ==== ==== ====    \n"""
"""   Col1 Col2 Col3\n"""
"""   ==== ==== ====  \n"""
"""    3    3.4  foo\n"""
"""    1    4.5  bar\n"""
"""   ==== ==== ====  \n""")
```
or perhaps better
```
    table = "\n".join([
"",
"# comment (with blank line above)",
"   ==== ==== ====    ",
"   Col1 Col2 Col3",
"   ==== ==== ====  ",
"    3    3.4  foo",
"    1    4.5  bar",
"   ==== ==== ====  ",
""])
```